webapp/components/wpt-results.html

```diff
@@ -372,13 +377,13 @@
       }
 
       async fetchManifest(path) {
-        if (!this.computePathIsATestFile(path)) {
-          return;
-        }
-        this.manifest = null; // Clear ASAP.
         this.manifest = await fetch(`/api/manifest?path=${path}`).then(r => r.json());
```

Comment:
Before the manifest was used only in `computeSourcePath` and `computeTestCanBeRunOnW3C` which made a revision mismatch prettty harmless. Now, https://manifest-test-list-dot-wptdashboard-staging.appspot.com/results/?sha=82bf967918 ends up looking quite different to https://wpt.fyi/results/?sha=82bf967918, including async-local-storage and other directories which didn't exist at the time.

How to fix? I guess the only way is to include the sha here, and do something (what?) if showing mixed results involving multiple wpt shas.

Reply:
I'll follow up this PR with forwarding the SHA to the manifest request (since we now use complete runs).

Reply:
Complete runs yay! Not deploying this change to https://wpt.fyi/results/?sha=82bf967918 before that's fixed would be good, since it'll make a mess of existing links. And that does require handling the case where the manifest isn't available, but maybe that's already fine?